containenv/containenv/cli/commands/init.py

```diff
@@ -48,3 +77,152 @@ def __call__(self):
         return run_tasks(
             'running some tasks',
             self.tasks)
+
+    def _make_task(self, method):
+        '''create a run_tasks compliant task'''
+        return (method.__doc__, method)
```

Comment:
But not so concerned that I think we should block this PR on it.   This is tracked in issue #29.